### PR TITLE
Rating page improvements

### PR DESCRIPTION
**Pull request by emilyb7**
waiting for #176  to be merged first

### CODE REVIEW

File: app/components/rate-goal/animations/cactus.jsx [hunk 1]
```diff
@@ -32,6 +32,8 @@ const createAnimation = ({target, options, }) => {
     { visibility: 'visible', },
   ];
 
+<<<<<<< HEAD
```
Comment (des-des) [inline app/components/rate-goal/animations/cactus.jsx]: ops, does this compile?

Comment (des-des) [inline app/components/rate-goal/animations/cactus.jsx]: Are you running your linter as a precommit hook?

Comment (emilyb7) [inline app/components/rate-goal/animations/cactus.jsx]: 🙀

Comment (emilyb7) [inline app/components/rate-goal/animations/cactus.jsx]: something went wrong with pushing to github i think, it was ok on my local branch. weird.

Comment (des-des) [inline app/components/rate-goal/animations/cactus.jsx]: prob was fixed but did not get commited?

**Pull request merged**